selenium-webdriver/chrome.d.ts

```diff
@@ -0,0 +1,418 @@
+import * as webdriver from "./index";
+import * as remote from "./remote";
+
+declare namespace chrome {
```

Comment:
So if a user wanted to do something like:

```
import { Driver as ChromeDriver } from 'selenium-webdriver/chrome';
```

This will no longer work?